prac_05/emails.py

```diff
@@ -0,0 +1,15 @@
+def main():
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        parts = email.split("@")
```

Comment:
Shouldn't the code to extract the name from the email be in a separate function?